chart/templates/networkpolicy.yaml

```diff
@@ -95,6 +95,12 @@ spec:
           podSelector:
             matchLabels:
               k8s-app: kube-dns
+        {{- if .Values.policies.networkPolicy.platformEnabled }}
```

Comment:
We want to toggle this separately and this shouldn't be just hard-coded like we did with dns when `.Values.policies.networkPolicy.enabled` is set to true?!